bin/publish-modules

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var semver = require( 'semver' );
+
+console.log( 'Modules requiring publish:' );
```

Comment:
Resolved in: https://github.com/belcherj/wp-calypso/commit/93e74a1c4f23dd2b60d42beb147367e10f1bf283